tests/core/data_storage/test_on_disk_storage.py

```diff
@@ -0,0 +1,174 @@
+import functools
+
+import numpy
+
+from openfisca_core import data_storage
+from openfisca_core import periods
+
+import pytest
+
+
+@pytest.fixture
+def storage():
+    return functools.partial(data_storage.OnDiskStorage, storage_dir = "/tmp")
```

Comment:
That style of import isn't an advantage when dealing with classes. Using
```
from openfisca_core.data_storage import OnDiskStorage
```
is just fine here.

The meta-rule for imports is, I should be able to look at the name of a method or class and if I'm not familiar with it, glance back to the imports section of the source file and immediately see where it is imported from. This saves a lot of time when debugging, compared to wildcard imports.

Systematically prefixing can be advantageous in particular cases like Numpy methods, but I wouldn't make it a general rule. It feels burdensome.

Reply:
It feels ok to me, although I'll stick to current convention for this library, wasn't intended, thanks for the pointer.

Reply:
Changed !